src/App.tsx

```diff
@@ -51,14 +50,14 @@ const App: React.FC = () => {
 
   const saveDesign = () => {
     // eslint-disable-next-line
-    emailEditorRef.current.editor.saveDesign((design: any) => {
+        emailEditorRef.current.editor.saveDesign((design: any) => {
```

Comment:
Me parece que no van esos espacios extra, puede ser?

Reply:
The format occurred because the line eslint-ignore ignore the rules and was fix typing the correct type object.
![image](https://user-images.githubusercontent.com/6733401/132023235-d0222b1e-b516-49dc-b353-3fe0e4e96b9b.png)